promotion/claim.go

```diff
@@ -154,14 +153,7 @@ func (service *Service) ClaimPromotionForWallet(
 		return nil, err
 	}
 
-	go func() {
-		err := service.datastore.RunNextClaimJob(ctx, service)
-		// FIXME
-		if err != nil {
-			logger := log.Ctx(ctx)
-			logger.Error().Err(err).Msg("error processing claim job")
-		}
-	}()
+	go service.CheckJobs(ctx, false)
```

Comment:
I don't see any reason to force this through `CheckJobs` - it seems simpler to duplicate 3 lines to log the error to sentry